application-admintools-api/src/main/java/com/xwiki/admintools/WikiSizeResult.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package com.xwiki.admintools;
+
+import org.xwiki.stability.Unstable;
+
+/**
+ * Stores info about the size of a Wiki.
+ *
+ * @version $Id$
+ * @since 1.0
+ */
+@Unstable
+public class WikiSizeResult
+{
+    private String wikiName;
+
+    private Long numberOfUsers;
+
+    private String attachmentSize;
+
+    private Long numberOfAttachments;
+
+    private Long numberOfDocuments;
+
+    /**
+     * Null constructor to initialize a {@link WikiSizeResult} object.
+     */
+    public WikiSizeResult()
+    {
+    }
+
+    /**
+     * Get the name of the Wiki.
```

Comment:
`.. name of the wiki.` 
Wiki is not a proper name to be used with upper case, as XWiki is. So lower case should be used
Below as well